docs/src/donu.md

```diff
@@ -475,4 +476,300 @@ Example:
     "type": "easel"
   }
 }
-```
\ No newline at end of file
+```
+
+### `stratify-command` - Stratify a model
+
+**Request**
+
+| Field               | Type                     | Description                                                                                       |
+|---------------------|--------------------------|---------------------------------------------------------------------------------------------------|
+| command             | string                   | Command - for this operation it will be the string `"stratify-command"`                           |
+| definition          | datasource               | The specification of the model                                                                    |
+| connection-graph    | datasource               | JSON connection graph specifying stratification pattern                                           |
+| state-metadata      | datasource               | JSON metadata describing infectious states (optional)                                             |
+| stratification-type | string                   | The type of stratification to perform. The value should be either `"spatial"` or `"demographic"`. |
+
+Example:
+
+```JSON
+command = {
+  "command": "stratify-command",
+  "definition": "model SIR:\n  let beta = 0.4\n  let gamma = 0.04\n\n  let s_initial = 997...",
+  "connection-graph": {
+    "file": "/home/rscott/Documents/Hacking/Haskell/ASKE-E/jupyter/jupyter2/connectionGraph/cities.txt"
+  },
+  "stratification-type": "spatial"
+}
+```
+
+**Response**
+
+| Field            | Type                     | Description                                                 |
+|------------------|--------------------------|-------------------------------------------------------------|
+| raw-model        | string                   | The stratified model (inline)                               |
+| pretty-model     | string                   | The stratified model, with some cosmetic touch-ups (inline) |
+| topology         | dictionary               | The topology                                                |
```

Comment:
I'm not quite sure how to describe this one. Any suggestions would be greatly appreciated.